examples/securenn/network_a.py

```diff
@@ -25,14 +26,18 @@
 
 class ModelTrainer(tfe.io.InputProvider):
 
-    BATCH_SIZE = 30
+    BATCH_SIZE = 32
```

Comment:
fyi 30 was picked to make sure all batches have the same size (ie divisor of 60k), not sure if that's a real problem

Reply:
nope! batch size affects the variance of the update, but not its expectation -- so having one batch be slightly more variant than the rest doesn't introduce any bias into the learning update